omnibus/Gemfile.lock

```diff
@@ -286,7 +286,7 @@ GEM
     mixlib-versioning (1.2.12)
     molinillo (0.8.0)
     multi_json (1.15.0)
-    multipart-post (2.1.1)
+    multipart-post (2.2.0)
```

Comment:
Based on the comment/lock above, this should still be 2.1.1

Reply:
Solved

Reply:
We updated license scout to find the license in the correct spot and move forward with the latest version of the multi-post gem.